projectq/backends/_awsbraket/_awsbraket_boto3_client.py

```diff
@@ -0,0 +1,502 @@
+            taskArn (str): The Arn of the task
+
+
+        """
+        argument = {
+            'circ': info['circuit'],
+            's3_folder': self._s3_folder,
+            'shots': info['shots'],
+        }
+
+        credentials = self._credentials
+        region_name = self.backends[device]['location']
+        deviceParameters = {
+            'braketSchemaHeader': self.backends[device]['deviceParameters'],
+            'paradigmParameters': {
+                'braketSchemaHeader':
+                    self.backends[device]['deviceModelParameters'],
+                'qubitCount': info['nq'],
+                'disableQubitRewiring': False,
+            },
+        }
+        deviceParameters = json.dumps(deviceParameters)
+
+        client_braket = boto3.client('braket', region_name=region_name,
+                    aws_access_key_id=self._credentials[0],
+                    aws_secret_access_key=self._credentials[1])
+
+        deviceArn = self.backends[device]['deviceArn']
+        response = client_braket.create_quantum_task(
+            action=argument['circ'],
+            deviceArn=deviceArn,
+            deviceParameters=deviceParameters,
+            outputS3Bucket=argument['s3_folder'][0],
+            outputS3KeyPrefix=argument['s3_folder'][1],
+            shots=argument['shots'])
+        taskArn = response['quantumTaskArn']
+
+        return taskArn
+
+    def _calculate_measurementProbs(measurements):
+        """
+        Calculate the measurement probabilities based on the list
+        of measurements for a job sent to a SV1 Braket simulator
+
+        Args:
+            measurements (list): list of measurements
+
+        Returns:
+            measurementsProbabilities (dict): The measurements
+            with their probabilities
+        """
+        total_mes = len(measurements)
+        unique_mes = [list(x) for x in set(tuple(x) for x in measurements)]
+        total_unique_mes = len(unique_mes)
+        len_qubits = len(unique_mes[0])
+        measurementsProbabilities = {}
+        for i in range(total_unique_mes):
+            strqubits = ''
+            for nq in range(len_qubits):
+                strqubits += str(unique_mes[i][nq])
+            prob = measurements.count(unique_mes[i])/total_mes
+            measurementsProbabilities[strqubits] = prob
+
+        return measurementsProbabilities
+
+    def _get_result(self,
+                    execution_id,
+                    num_retries=30,
+                    interval=1,
+                    verbose=False):
+
+        if verbose:
+            print("Waiting for results. [Job Arn: {}]".format(execution_id))
+
+        original_sigint_handler = signal.getsignal(signal.SIGINT)
+
+        def _handle_sigint_during_get_result(*_):  # pragma: no cover
+            raise Exception(
+                "Interrupted. The Arn of your submitted job is {}.".format(
+                    execution_id))
+
+        def _calculate_measurementProbs(measurements):
```

Comment:
What is the difference between this nested function and the one defined on line 231?

Reply:
No difference, just a copy-paste error :-) . The correct is the nested one (inside _get_result). I deleted the extra function